public/scripts/controllers/navigation.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+(function(module) {
+  const navigation = {};
+
+  navigation.loadIndexPage = function() {
+    console.log('loading the home page');
+    $('.tab-content').hide();
+    $('#home').fadeIn(600);
+    $('#top-nav').hide(); // fade in has a weird ghost effect
+
```

Comment:
I tried this using the fadeIn, as you use elsewhere and it seems fine to me. It's seems simply to be a case of what you prefer to see on the home page: nav bar or no nav bar. I think it's fine either way.